contribs/gmf/less/mobilelayertree.less

```diff
@@ -21,4 +21,8 @@ nav.nav-left .gmf-layertree-node a[data-toggle] {
   .group {
     font-size: @font-size-base;
   }
+
+  [ngeo-popover] {
+    display: none;
+  }
```

Comment:
Not really I just found the problem while I was testing, the popover should not be display on mobile right?